chia/wallet/cat_wallet/lineage_store.py

```diff
@@ -19,7 +19,7 @@ class CATLineageStore:
     table_name: str
 
     @classmethod
-    async def create(cls, db_wrapper: DBWrapper2, asset_id: str):
+    async def create(cls, db_wrapper: DBWrapper2, asset_id: str) -> "CATLineageStore":
```

Comment:
```suggestion
    async def create(cls, db_wrapper: DBWrapper2, asset_id: str) -> CATLineageStore:
```

Reply:
this change breaks flake8. `CATLineageStore` is not defined yet at this point

Reply:
`from __future__ import annotations`

Reply:
I see. I looked in the wrong file, I thought we had this already